vyper/context/types/function.py

```diff
@@ -26,6 +27,34 @@
 )
 
 
+class FunctionVisibility(StringEnum):
+    PUBLIC = StringEnum.auto()
+    PRIVATE = StringEnum.auto()
+
+
+class StateMutability(StringEnum):
+    PURE = StringEnum.auto()
+    VIEW = StringEnum.auto()
+    NONPAYABLE = StringEnum.auto()
+    PAYABLE = StringEnum.auto()
+
+    @classmethod
+    def from_abi(cls, abi_dict) -> "StateMutability":
+        """
+        Extract stateMutability from an entry in a contract's ABI
+        """
+        if "stateMutability" in abi_dict:
+            return cls(abi_dict["stateMutability"])
+        elif "payable" in abi_dict and abi_dict["payable"]:
```

Comment:
```suggestion
        elif abi_dict.get("payable"):
```

could do this for constant as well..  bit less verbose.

Reply:
Shouldn't it be?
```suggestion
        elif abi_dict.get("payable", False):
```

Reply:
`get` returns `None` if the key doesn't exist